libasn1compiler/asn1c_misc.c

```diff
@@ -61,8 +61,13 @@ asn1c_make_identifier(enum ami_flags_e flags, asn1p_expr_t *expr, ...) {
 		/*
 		 * Estimate the necessary storage size
 		 */
-		if(expr->Identifier == NULL)
-			return "Member";
+		if(expr->Identifier == NULL) {
+			if(storage_size < 10) {
+				storage = calloc(1, 20);  /* reserve space for 'Memberxxx' */
```

Comment:
This is a memory leak. Free `storage` first. You need to update `storage_size` as well as it is also static.

Reply:
Thanks, I meant to use realloc, but....

I will fix it.